benchmark/python/benchmark_hail/run/linalg_benchmarks.py

```diff
@@ -15,3 +15,18 @@ def make_ndarray_bench():
     ht = hl.utils.range_table(200_000)
     ht = ht.annotate(x=hl._nd.array(hl.range(200_000)))
     ht._force_count()
+
+@benchmark()
+def ndarray_addition_benchmark():
+    arr = hl._nd.ones((1024, 1024))
+    hl.eval(arr + arr)
```

Comment:
this is going to be testing stuff other than addition, primarily -- like sending the result back to python. That seems fine for now.